tests/invalid_models_tests/test_deprecated_fields.py

```diff
@@ -21,3 +22,21 @@ class IPAddressModel(models.Model):
                 id='fields.E900',
             )],
         )
+
+    def test_CommaSeparatedIntegerField_deprecated(self):
+        class CommaSeparatedIntegerModel(models.Model):
+            csi = models.CommaSeparatedIntegerField(max_length=64)
+
+        model = CommaSeparatedIntegerModel()
+        self.assertEqual(
+            model.check(),
+            [checks.Warning(
+                'CommaSeparatedIntegerField has been deprecated. Support '
+                'for it (except in historical migrations) will be removed'
+                ' in Django 2.0',
+                hint=('Use CharField(validators='
```

Comment:
single longer line is okay (we prefer up to 119 characters when it helps readability).